server/src/routes/UserController.ts

```diff
@@ -0,0 +1,63 @@
+import { addUser, getUsers, deleteUser, updateUser } from "#";
```

Comment:
What is this dependency? I get the following error in code:

Cannot find module '#' or its corresponding type declarations.ts(2307)